modules/images/webp-uploads/load.php

```diff
@@ -573,3 +562,52 @@ function webp_uploads_update_featured_image( $html, $post_id, $attachment_id ) {
 	return webp_uploads_img_tag_update_mime_type( $html, 'post_thumbnail_html', $attachment_id );
 }
 add_filter( 'post_thumbnail_html', 'webp_uploads_update_featured_image', 10, 3 );
+
+/**
+ * Updates the image src data to use the webp format if an image has webp version.
+ *
+ * @since n.e.x.t
+ *
+ * @param array|bool   $image         Image data.
+ * @param int          $attachment_id Image attachment ID.
+ * @param string|array $size          Requested image size.
```

Comment:
We should match the doc block written for WordPress: https://developer.wordpress.org/reference/hooks/wp_get_attachment_image_src/

Reply:
Thanks, but we reverted this function.